src/app/api/_api-services/ai_service/index.ts

```
@@ -294,7 +296,8 @@ export class AIService {
 		postIndexOrHash: string;
 		comments: ICommentResponse[];
 	}): Promise<string | null> {
-		if (!comments?.length) {
+		// Require minimum comments before generating summary
+		if (!comments?.length || comments.length < this.MIN_COMMENTS_FOR_SUMMARY) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)